elytron-oidc-client/subsystem/src/main/resources/org/wildfly/extension/elytron/oidc/LocalDescriptions.properties

```diff
@@ -1,144 +1,142 @@
+elytron-oidc-client.secure-deployment.cors-exposed-headers=If the Cross-Origin Resource Sharing (CORS) is enabled, this sets the value of the 'Access-Control-Expose-Headers' header. This should be a comma-separated string. This is optional. If not set, this header is not returned in CORS responses.
+elytron-oidc-client.secure-deployment.expose-token=If set to 'true', an authenticated browser client can obtain the signed access token (through a Javascript HTTP invocation) via the URL 'root/k_query_bearer_token'. This is optional.
+elytron-oidc-client.secure-deployment.auth-server-url-for-backend-requests=The 'auth-server-url' URL used only for backend requests to the OpenID provider.
+elytron-oidc-client.secure-deployment.always-refresh-token=If set to 'true', the subsystem will refresh token on every web request to your application and a new request will be sent to the OpenID provider.
+elytron-oidc-client.secure-deployment.register-node-at-startup=If set to 'true', the subsystem sends a registration request to the OpenID provider. This attribute is useful only when your application is clustered.
+elytron-oidc-client.secure-deployment.register-node-period=If 'register-node-at-startup' is set to 'true', this defines how often to re-register the node.
+elytron-oidc-client.secure-deployment.token-store=Defines whether to store account information in an HTTP session or in a cookie.
+elytron-oidc-client.secure-deployment.principal-attribute=Indicates which value from the ID token to use as the principal for the identity. The principal defaults to the value of the "sub" if the token attribute is null.
+elytron-oidc-client.secure-deployment.turn-off-change-session-id-on-login=The Session ID is changed by default on a successful login. Set this to 'true' if you want to turn this off.
+elytron-oidc-client.secure-deployment.token-minimum-time-to-live=The subsystem will refresh the token if it will expire within the duration specified in 'token-minimum-time-to-live' seconds. This value should never exceed the access token lifespan. If set to 0 seconds, the subsystem will refresh the token only if expired.
+elytron-oidc-client.secure-deployment.min-time-between-jwks-requests=If the subsystem recognizes token signed by an unknown public key, it will try to download a new public key from the server. It won't try to download if already tried in less than 'min-time-between-jwks-requests' seconds.
+elytron-oidc-client.secure-deployment.public-key-cache-ttl=The maximum interval between two requests to retrieve new public keys. New public keys are downloaded when the subsystem recognizes token signed by an unknown public key, but at least one per interval set in 'public-key-cache-ttl' new public keys are downloaded even if the token's key is already known.
```

Comment:
How about:

"The maximum interval between two requests to retrieve new public keys. New public keys are downloaded when the subsystem recognizes a token signed by an unknown public key. Even if the token's key is already known, new public keys are downloaded periodically as per the interval set here."